aries-site/src/data/wcag/rangeinput.json

```diff
@@ -0,0 +1,512 @@
+    "notes": ""
+  },
+  {
+    "rule": "2.4.11",
+    "label": "Focus Not Obscured (Minimum)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.12",
+    "label": "Focus Not Obscured (Enhanced)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.13",
+    "label": "Focus Appearance",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.1",
+    "label": "Pointer Gestures",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.2",
+    "label": "Pointer Cancellation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.3",
+    "label": "Label in Name",
+    "status": "conditional",
+    "notes": "Screen reader must read the label of the range input"
+  },
+  {
+    "rule": "2.5.4",
+    "label": "Motion Actuation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.5",
+    "label": "Target Size (Enhanced)",
+    "status": "failed",
+    "notes": "Fails AAA but passes AA"
+  },
+  {
+    "rule": "2.5.6",
+    "label": "Concurrent Input Mechanisms",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.7",
+    "label": "Dragging Movements",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.8",
+    "label": "Target Size (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.1",
+    "label": "Language of Page",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.2",
+    "label": "Language of Parts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.3",
+    "label": "Unusual Words",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.4",
+    "label": "Abbreviations",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.5",
+    "label": "Reading Level",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.6",
+    "label": "Pronunciation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.1",
+    "label": "On Focus",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.2",
+    "label": "On Input",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.3",
+    "label": "Consistent Navigation",
+    "status": "conditional",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.4",
+    "label": "Consistent Identification",
+    "status": "conditional",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.5",
+    "label": "Change on Request",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.6",
+    "label": "Consistent Help",
+    "status": "conditional",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.1",
+    "label": "Error Identification",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField"
+  },
+  {
+    "rule": "3.3.2",
+    "label": "Labels or Instructions",
+    "status": "not-applicable",
```

Comment:
I would change to conditional

Reply:
@jcfilben what do you think for these since they would apply only if wrapped in formfield?

Reply:
yeah the only reason why I would consider `conditional` is because we kinda recommend our inputs be used with formfields but I also see that this would the other side of this being not applicable since it not directly tied with RangeInput so I could go either way

Reply:
chatted with Jessica leave as not-applicable